src/webapp/logic/sheetBuilder.ts

```diff
@@ -825,8 +825,33 @@ export class SheetBuilder {
         );
 
         if (element.type === "programs") {
-            this.createColumn(dataEntrySheet, itemRow, columnId++, i18n.t("Latitude", { lng: this.builder.language }));
-            this.createColumn(dataEntrySheet, itemRow, columnId++, i18n.t("Longitude", { lng: this.builder.language }));
+            const { element: program } = this.builder;
+            const featureType = _(program.programStages)
+                .map(ps => ps.featureType)
+                .first();
+
+            if (featureType === "POLYGON") {
+                this.createColumn(
+                    dataEntrySheet,
+                    itemRow,
+                    columnId++,
+                    i18n.t("Polygon in map\n([[LON1, LAT1], [LON2, LAT2], ...])", { lng: this.builder.language })
```

Comment:
I was going to point out that the everyday format of coordinates is [LAT, LON] (We have [LON, LAT] because that's what DHIS2 uses internally). But being a polygon, so a generated thing, it's probably not so important to the final user as long as we state the required order (which we do). So nothing to change.